pages/index.qmd

```diff
@@ -1,18 +1,31 @@
 ---
 title: "Introduction"
+include-after-body:
+    file: includes/after-body.html
 ---
 
 
 
-The United States SARS-CoV-2 Variant Nowcast Hub has been designed by researchers from the [US CDC Center of Forecasting and Outbreak Analytics (CFA)](https://www.cdc.gov/forecast-outbreak-analytics/index.html) and [the Reich Lab at UMass Amherst](https://reichlab.io/), in consultation with folks from [the NextStrain project](https://nextstrain.org/).
+The United States SARS-CoV-2 Variant Nowcast Hub has been designed by researchers from the [US CDC Center of Forecasting and Outbreak Analytics (CFA)](https://www.cdc.gov/forecast-outbreak-analytics/index.html) and [the Reich Lab at UMass Amherst](https://reichlab.io/), in consultation with [the NextStrain project](https://nextstrain.org/) team.
 
 Teams may submit predictions of the prevalence of the predominant SARS-CoV-2 clades in the US, at a daily timescale and the geographic resolution of all 50 United States plus Washington, DC and Puerto Rico.
 
-### Predicted clades
+## Nextstrain clades and pango lineages
+
+[Nextstrain clade names](https://docs.nextstrain.org/projects/ncov/en/latest/reference/naming_clades.html), such as "19A" or "24H", "label genetically well defined clades that have reached significant frequency and geographic spread". The Variant Nowcast Hub use Nextstrain clades, although model outputs, as shown in the model reports, are translated so that clades are matched with pango lineages. One reason the hub chose to use the Nextstrain clades as the grouping for the modeling is that the broader "clade" grouping allows for a smaller number of categories to be modeled in any given week compared with pango lineages, which do not have a similar high-level grouping.
+
+Nextstrain provides [an interactive phylogenetic tree for SARS-CoV-2](https://nextstrain.org/nextclade/sars-cov-2), with Nextstrain clade labels annotating branches of the tree, and each sequence receiving both a pango lineage and Nextstrain clade assignment.
+
+The following list provides a partial history of Nextstrain clade announcements, which include detailed discussion about the origin and genetic differences with other lineages and clades: 
+
+- [24H (LF.7) and 24I (MV.1)](https://github.com/nextstrain/ncov/pull/1158) (18 Oct 2024)
+- [24F (XEC) and 24G (KP.2.3)](https://github.com/nextstrain/ncov/pull/1152) (25 Sep 2024)
+
```

Comment:
Is the plan to continue to update this with a list of PRs pertaining to the mappings from nextclade clades to pangolineages?

Reply:
yes, I was imagining we would manually update when we notice that new lineages are out (and we are using them in our nowcasts).